tahoe_sites/models.py

```diff
@@ -17,7 +17,7 @@ class UserOrganizationMapping(models.Model):
 
     Tahoe's fundamental multi-site relationship.
     """
-    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='+')
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='+')
```

Comment:
**Required change:** This is not consistent with `edx-organizations`. We need to do either of the following:

 1. Reflect on `edx-organizations`
     - Clean up production and staging data from duplicate
     - Change the field to be `OneToOneField` in `edx-organizations` as well
 2. Revert the change from this repo, and only make it when we're using the upstream `edx-organizaitons` and the migration is complete

Both options are fine, please pick one of them. Having inconsistent tables between `edx-organizations` and `tahoe-sites` would complicate our efforts to make a zero-downtime deployment.

Reply:
Thank you @OmarIthawi  for your reviews. I've reverted the change, and added check on `save`. Please review tests in `TestUserOrganizationMapping` . I made it work for both `False/True` for `TAHOE_SITES_USE_ORGS_MODELS`

Reply:
https://github.com/appsembler/tahoe-sites/pull/25